docs/docs/tutorials/todos.md

```diff
@@ -0,0 +1,665 @@
+---
+sidebar_position: 4
+description: Build a simple "Todos" application.
+---
+
+# Todos 🗒
+
+:::info
+**Difficulty**: 🟠 Intermediate<br/>
+**Length**: 30 minutes
+
+Before getting started, [read the prerequisites](/docs/overview#prerequisites) to make sure your development environment is ready.
+:::
+
+## Overview
+
+In this tutorial, we're going to build an app that exposes two endpoints which allow us to perform C.R.U.D operations on a list of todos.
```

Comment:
Never seen CRUD being referenced like that, with the dots.

I always use CRUD, as defined by [wiki](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)

Reply:
Good point, will replace